tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -47,4 +55,49 @@ public function emptyValueProvider()
             'false' => [false, true]
         ];
     }
+
+    /**
+     * @dataProvider priceValueProvider
+     *
+     * @param $value string|int value that should be checked.
+     * @param $shouldCauseException bool should an exception be caused by given parameter.
+     */
+    public function testPriceValueDetectsNumericsOnly($value, $shouldCauseException)
```

Comment:
Rename test and code as it should test the `UsergroupAwareNumericValue` class and not a specific data element.

Reply:
✅